homeassistant/components/wled/helpers.py

```diff
@@ -35,3 +35,13 @@ async def handler(self: _WLEDEntityT, *args: _P.args, **kwargs: _P.kwargs) -> No
             raise HomeAssistantError("Invalid response from WLED API") from error
 
     return handler
+
+
+def kelvinTo255(k: int, minK: int, maxK: int) -> int:
```

Comment:
Rename these to be Pythonic

```suggestion
def kelvin_to_255(k: int, min_k: int, max_k: int) -> int:
```